keps/sig-node/3695-pod-resources-for-dra/README.md

```diff
@@ -265,6 +265,12 @@ These cases will be added in the existing e2e tests:
   - Get API work with DRA and device plugin.
   - List API work with DRA and Device plugin.
 
+[Get](https://github.com/kubernetes/kubernetes/pull/116846): [sig-node-kubelet](https://testgrid.k8s.io/sig-node-kubelet?include-filter-by-regex=PodResources), [triage](https://storage.googleapis.com/k8s-triage/index.html?test=PodResources)
+
+Other improvements are addressed in:
+- https://github.com/kubernetes/kubernetes/pull/132028
+- https://github.com/kubernetes/kubernetes/pull/132345
+
```

Comment:
I think, but please doublecheck, is better not to link PRs in the KEP but rather describe the goal these PR implement.
The `List` fix I mentioned previously is meant to illustrate 
1. the very case we missed
2. the problem with API underspecification - which we kinda have for `Get`.

So I'd just remove lines 270-272